database/migrations/sqls/20220127232724-AddCapturedAtToCapture-up.sql

```diff
@@ -1,2 +1,4 @@
 /* Replace with your SQL commands */
-ALTER TABLE capture ADD COLUMN captured_at timestamp with time zone NOT NULL;
```

Comment:
I end up getting an error when running this. I guess db complains about creating this column with null values for existing records. This may not be an issue in prod db but is an issue in dev db. I also wonder how we got this column added via db-migrate, I get an error when run without the following changes.

Reply:
Maybe it wasn't updated via db-migrate? Because you are right it would have thrown an error @ZavenArra